### DISCUSSION TITLE
Attempting to link with myself

### DISCUSSION THREAD

**Discussion by SkyO-Steen** [visualboyadvance-m/visualboyadvance-m#1376] [Q&A]
Hi! I am a YouTuber trying to record a playthrough series of both Pokemon Red and Blue using the emulator. Everything is working perfectly fine, but I was wondering if there was a way to link two open windows of the emulator to each other on my one computer so I could trade Pokemon between the two versions I wouldn't be able to get otherwise. If it's possible, I'd love to know how, and if it's not possible on just one computer, could I please get an alternative? All help is appreciated.

**Comment by Squall-Leonhart** [comment]
Currently, no.

DMG linking was broken between 1.7l and 1.72l

**Reply by SkyO-Steen** [reply]
Okay, is there a possible alternative? I do have another laptop, if I put VBA on that would I be able to do a LAN link between those, or is it all entirely broken?
